### ISSUE TITLE
HUD Filtering

### ISSUE THREAD

**Issue by moonsidefan** [Mr-Wiseguy/Zelda64Recomp#53]
Some elements of the HUD (such as ActionButton symbols like A, B, Cs or sun and moon sprites rotating around the main timer display) have a smoothing filter while elements like the rupie count or the map do not have one. The emulator applies a smoothing filter to almost everthing HUD related I think. 

Unfortunately I dont know what the correct behaviour of the HUD filtering should be. Is the one of this project the right one?

**Comment by NineKain** [comment]
x2, I think there should be a way to choose the HUD filtering, at least until custom textures are supported

**Comment by mrpenguinb** [comment]
Yeah, not all of the elements are filtered which gives an inconsistent look :(

**Comment by alby13** [comment]
i did notice that the rupee HUD element was glaringly obvious with the pixelation. i was like, what's up with that?
